src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css

```diff
@@ -46699,6 +46714,7 @@ a[dir]:root[data-theme=darkmode] .badge-dark:focus, a[dir]:root[data-theme=darkm
 }
 :root[data-theme=darkmode] .leftbar-compactor:before {
   content: "⟪";
+  font-size: initial;
```

Comment:
Why is this necessary?

Reply:
The ⟪ would be invisible too otherwise. But I will try to find another way to hide the text.

Reply:
What about RTL? coz the direction is matter

Reply:
Does RTL matter here? The going to be text hidden anyway, won't it?

Reply:
Does RTL matter here? The text going to be hidden anyway, won't it?

Reply:
I though the arrows will be shown, while it's in the content

Reply:
The `font-size` or other changes of Uchral here don't affect RTL.